src/wasm-delegations-fields.def

```diff
@@ -461,12 +461,16 @@ switch (DELEGATE_ID) {
   }
   case Expression::Id::MemorySizeId: {
     DELEGATE_START(MemorySize);
+    DELEGATE_FIELD_TYPE(MemorySize, type);
```

Comment:
`type` is not needed in both cases, as it is present in all these classes and handled automatically.

Reply:
Great, thanks!